--- /docker/features/_masterfiles/5.2.1/overleaf/services/web/app/src/Features/Project/ProjectListController.js	2024-12-11 20:32:32.905963857 +0000
+++ ../5.2.1/overleaf/services/web/app/src/Features/Project/ProjectListController.js	2024-12-10 17:43:53.113910424 +0000
@@ -167,17 +167,7 @@
     if (user && UserPrimaryEmailCheckHandler.requiresPrimaryEmailCheck(user)) {
       return res.redirect('/user/emails/primary-email-check')
     }
-  } else {
-    if (!process.env.OVERLEAF_IS_SERVER_PRO) {
-      // temporary survey for CE: https://github.com/overleaf/internal/issues/19710
-      survey = {
-        name: 'ce-survey',
-        preText: 'Help us improve Overleaf',
-        linkText: 'by filling out this quick survey',
-        url: 'https://docs.google.com/forms/d/e/1FAIpQLSdPAS-731yaLOvRM8HW7j6gVeOpcmB_X5A5qwgNJT7Oj09lLA/viewform?usp=sf_link',
-      }
-    }
-  }
+  } 
 
   const tags = await TagsHandler.promises.getAllTags(userId)
 
