--- /docker/features/_masterfiles/5.2.1/overleaf/services/web/config/settings.defaults.js	2024-12-13 00:24:53.563269274 +0000
+++ ../5.2.1/overleaf/services/web/config/settings.defaults.js	2024-12-13 00:24:29.332558249 +0000
@@ -259,6 +259,9 @@
     notifications: {
       url: `http://${process.env.NOTIFICATIONS_HOST || '127.0.0.1'}:3042`,
     },
+    references: {
+      url: `http://${process.env.REFERENCES_HOST || '127.0.0.1'}:3056`,
+    },
     webpack: {
       url: `http://${process.env.WEBPACK_HOST || '127.0.0.1'}:3808`,
     },
