--- /docker/features/_masterfiles/5.2.1/overleaf/services/web/app/src/infrastructure/Features.js	2024-12-11 19:54:34.900262205 +0000
+++ ../5.2.1/overleaf/services/web/app/src/infrastructure/Features.js	2024-12-08 16:02:12.465592480 +0000
@@ -35,6 +35,7 @@
     return (
       (Boolean(Settings.ldap) && Boolean(Settings.ldap.enable)) ||
       (Boolean(Settings.saml) && Boolean(Settings.saml.enable)) ||
+      (Boolean(Settings.oidc) && Boolean(Settings.oidc.enable)) ||
       Boolean(Settings.overleaf)
     )
   },
