--- /docker/features/_masterfiles/5.2.1/overleaf/services/web/app/views/layout/navbar-marketing-bootstrap-5.pug	2024-12-11 19:54:41.763180193 +0000
+++ ../5.2.1/overleaf/services/web/app/views/layout/navbar-marketing-bootstrap-5.pug	2024-12-08 16:02:12.467592456 +0000
@@ -136,14 +136,24 @@
 
 						// login link
 						+nav-item
-							+nav-link(
-								href="/login"
-								event-tracking="menu-clicked-login"
-								event-tracking-action="clicked"
-								event-tracking-trigger="click"
-								event-tracking-mb="true"
-								event-segmentation={ page: currentUrl }
-							) #{translate('log_in')}
+							if settings.oidc.enable
+								+nav-link(
+									href="/login/oidc"
+									event-tracking="menu-clicked-login"
+									event-tracking-action="clicked"
+									event-tracking-trigger="click"
+									event-tracking-mb="true"
+									event-segmentation={ page: currentUrl }
+								) #{translate('login_oidc', {provider: settings.oidc.nameShort})}
+							else
+								+nav-link(
+									href="/login"
+									event-tracking="menu-clicked-login"
+									event-tracking-action="clicked"
+									event-tracking-trigger="click"
+									event-tracking-mb="true"
+									event-segmentation={ page: currentUrl }
+								) #{translate('log_in')}
 
 					// projects link and account menu
 					if getSessionUser()
