--- /docker/features/_masterfiles/5.2.1/overleaf/services/web/app/src/infrastructure/Features.js	2024-12-15 02:57:40.316087670 +0000
+++ ../5.2.1/overleaf/services/web/app/src/infrastructure/Features.js	2024-12-15 02:54:53.905061361 +0000
@@ -52,10 +52,7 @@
       case 'homepage':
         return Boolean(Settings.enableHomepage)
       case 'registration-page':
-        return (
-          !Features.externalAuthenticationSystemUsed() ||
-          Boolean(Settings.overleaf)
-        )
+        return Boolean(true)
       case 'registration':
         return Boolean(Settings.overleaf)
       case 'chat':
