--- /docker/features/_masterfiles/5.2.1/overleaf/services/web/app/src/Features/User/UserPrimaryEmailCheckHandler.js	2024-12-11 19:54:25.731371773 +0000
+++ ../5.2.1/overleaf/services/web/app/src/Features/User/UserPrimaryEmailCheckHandler.js	2024-12-08 16:02:12.465592480 +0000
@@ -6,6 +6,11 @@
   lastPrimaryEmailCheck,
   signUpDate,
 }) {
+  if(Settings.oidc.enable) {
+    // we never require a check, as emails are retrieved from the OIDC provider
+    return false
+  }
+
   const hasExpired = date => {
     if (!date) {
       return true
